eng/xunit.runner.json

```diff
@@ -1,3 +1,3 @@
 {
-  "shadowCopy": false,
+  "shadowCopy": false
```

Comment:
Looks like @chidozieononiwu just merged in this change as well so you can undo this to resolve the conflict.

Reply:
Thanks for the heads-up;  I'll scrub this on rebase.